nni/compression/pytorch/compressor.py

```diff
@@ -433,6 +433,17 @@ def load_model_state_dict(self, model_state):
         else:
             self.bound_model.load_state_dict(model_state)
 
+    def get_pruned_weights(self, dim=0):
+        for wrapper_idx, wrapper in enumerate(self.get_modules_wrapper()):
+            weight_mask = wrapper.weight_mask
+            if len(weight_mask.size()) == 4:
+                sum_idx = (1, 2, 3) if dim == 0 else (0, 2, 3)
```

Comment:
```
sum_idx = [0, 1, 2, 3]
sum_idx.remove(dim)
```
or if we only handle `dim=0` and `dim=1`, we could use a `boolean` to mark it.

Reply:
Thanks, fix it.